Week-3/Assingment3.3/Assingment3.3.3/Assingment3.3.3.js

```diff
@@ -0,0 +1,22 @@
+function isStrongPassword(password) {
+    if (password.length < 8) {
+        return false;
+    }
+
+    if (password.includes("password")) {
+        return false;
+    }
+
+    let hasUppercase = false;
+    for (let char of password) {
+        if (char >= 'A' && char <= 'Z') {
+            hasUppercase = true;
```

Comment:
maybe you can try exploring regex to solve this

Reply:
done